frontend/src/components/AnnotationRenderer.vue

```diff
@@ -124,8 +126,23 @@ export default {
         return  getValueFromKeyPath(this.document, this.doc_preannotation_field)
       }
       return null
+    },
+    shownElements() {
+      if (!this.config) {
+        return [];
+      }
+      return this.config.filter((elemConfig) => {
+        if (this.condition[elemConfig.name]) {
+          try {
+            return !!this.condition[elemConfig.name].evalSync({document: this.document, annotation: this.annotationOutput});
```

Comment:
I've used `annotation` for the current annotation state in the expression bindings, but possibly a name like `values` might make more sense.